tests/console/console.go

```diff
@@ -226,7 +226,11 @@ func NewExpecter(
 	if err != nil {
 		return nil, nil, err
 	}
-	timeout -= time.Since(startTime)
+	serialConsoleCreateDuration := time.Since(startTime)
+	if timeout-serialConsoleCreateDuration <= 0 {
+		return nil, nil, fmt.Errorf("creation of SerialConsole took %s - longer than given expecter timeout %s", serialConsoleCreateDuration.String(), timeout.String())
```

Comment:
This broke the optional lint lane. 

Is there an opened PR about this ? @AlonaKaplan / @orelmisan

Reply:
This was fixed by @fossedihelm in #9515